viz.yaml

```diff
@@ -281,6 +282,10 @@ publish:
     id: map_counties
     location: "js/counties.js"
     mimetype: application/javascript
+  -
+    id: dropdown_selector
+    location: "js/dropdown_selector.js"
```

Comment:
Good idea to split this functionality up.  Was torn between doing that or just adding it to the map_utils, as it affects the map.  Probably smarter to keep it separated.